codice/ddf
# DDF-4384 relaxed GeoUtil unit test

#### What does this PR do?
Simplifying a point radius search to a polygon results in a very precise value for lat/lon coordinates. This causes the matchers to fail the `testConvertPointRadiusToCirclePolygon()` unit test, because of differences in OS & JVMs producing very slightly different values due to the processing of floating point precision. 

This PR relaxes the unit test to be more consistent.

#### Who is reviewing it? 
@coyotesqrl @ryeats @paouelle @willwill96 @gjvera 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@coyotesqrl

#### How should this be tested?
Since this change only affects a unit test, running that unit test should suffice.
Recommend running the unit test with with a variety of JVMs to ensure the test is relaxed enough.

#### Any background context you want to provide?

#### What are the relevant tickets?
[DDF-4384](https://codice.atlassian.net/browse/DDF-4384)

#### Screenshots

#### Checklist:
- [ X ] Update / Add Unit Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
